newsfragments/4250.added.md

```diff
@@ -0,0 +1 @@
+Implement `PartialEq<str>` for `Bound<'py, PyBytes>`.
```

Comment:
```suggestion
Implement `PartialEq<[u8]>` for `Bound<'py, PyBytes>`.
```

Reply:
I saw this but wasn't able to accept it in time before the merge.

Reply:
I'll fix it on my next PR for #4245

Reply:
No prob; I got it in #4038 thanks to @alex